snetd/cmd/components.go

```diff
@@ -33,6 +34,8 @@ type Components struct {
 	daemonHeartbeat            *metrics.DaemonHeartbeat
 	paymentStorage             *escrow.PaymentStorage
 	priceStrategy              *pricing.PricingStrategy
+	configurationService       *configuration_service.ConfigurationService
+	channelBroadcast           *configuration_service.MessageBroadcaster
```

Comment:
Why don't name it `configurationBroadcaster` or `configurationBroadcastingChannel`